README.rst

```diff
@@ -23,10 +23,10 @@
 .. |DOI| image:: https://zenodo.org/badge/21369/scikit-learn/scikit-learn.svg
 .. _DOI: https://zenodo.org/badge/latestdoi/21369/scikit-learn/scikit-learn
 
-scikit-learn
-============
+.. image:: doc/logos/scikit-learn-logo.png
+	:align: center
```

Comment:
don't use tabs, just spaces please

Reply:
ping @erinkhoo this is still a tab, please use spaces

Reply:
Updating now.